### PR TITLE
Feat/leverage updates

### PR DESCRIPTION
**Pull request by matthew-garrett**
- Adds Toggle to Portfolio Holdings table
- Adds ability for txn history table to be filtered by multiple assets
- Adds total row to holdings table
- Other bug fixes

### CODE REVIEW

File: packages/common/mui/src/lib/data-table/data-table-toggle/data-table-toggle.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+};
+
+const StyledTabs = styled(Tabs, {
+  shouldForwardProp: (prop: string) => prop !== 'minHeight',
+})(
+  ({ minHeight, theme }: StyledTabProps) => `
+  height: 100%;
+  border-radius: ${theme?.shape.borderRadius()};
+  border: ${theme?.shape.borderStandard};
+  max-height: 3rem;
```
Comment (jeffywu) [inline packages/common/mui/src/lib/data-table/data-table-toggle/data-table-toggle.tsx:59]: theme.spacing(6)

File: packages/common/mui/src/lib/trade-action-header/trade-action-header.tsx [hunk 2]
```diff
@@ -42,14 +41,14 @@ export function TradeActionHeader({
...
-      <H3
+      <Box
         sx={{
           fontWeight: theme.typography.fontWeightRegular,
           marginLeft: theme.spacing(2),
         }}
       >
         {actionText}
-      </H3>
+      </Box>
```
Comment (jeffywu) [inline packages/common/mui/src/lib/trade-action-header/trade-action-header.tsx:51]: Should this switch back to H3? Does the font size look right here?

File: packages/shared/trade/src/transaction-confirmation/components/status-heading.tsx [hunk 3]
```diff
@@ -77,7 +78,15 @@ export const StatusHeading = ({
           alt="Exclamation point"
         />
       );
-      headingText = <FormattedMessage defaultMessage={'Error'} />;
+      headingText = (
+        <Box
+          sx={{
+            color: colors.red,
```
Comment (jeffywu) [inline packages/shared/trade/src/transaction-confirmation/components/status-heading.tsx:84]: theme.palette.error.main i think....

File: packages/common/mui/src/lib/data-table/data-table-toggle/data-table-toggle.tsx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  }
+`
+);
+
+const StyledTab = styled(Tab)(
+  ({ theme }) => `
+  height: 100%;
+  font-family: ${theme.typography.fontFamily};
+  color: ${colors.greenGrey};
+  font-weight: 500;
```
Comment (jeffywu) [inline packages/common/mui/src/lib/data-table/data-table-toggle/data-table-toggle.tsx:87]: I think you can use theme.typography.fontWeight.regular here or something along those lines

File: packages/common/mui/src/lib/trade-action-header/trade-action-header.tsx (hunk 2 above)
Comment (matthew-garrett) [reply packages/common/mui/src/lib/trade-action-header/trade-action-header.tsx:51]: This was an intentional change. H3 does not look right.

**Comment by jeffywu** [comment]
Ok that’s cool

**Pull request merged**